src/MessagePack.UnityClient/Assets/Scripts/MessagePack/ThisLibraryExtensionTypeCodes.cs

```diff
@@ -58,6 +58,11 @@ internal static class ThisLibraryExtensionTypeCodes
         /// </summary>
         internal const sbyte UnityDouble = 39;
 
+        /// <summary>
+        /// The LZ4 compression extension.
```

Comment:
This deserves a distinct description from the `LZ4` one.

Reply:
Yes